Sapna2001/task5/Avatar generator/index.js

```diff
@@ -0,0 +1,7 @@
+$(document).ready(function(){
+ $('button').click(function(){
+  const name = $('#nameid').val();
+  $('.imgage').attr('src',`https://joeschmoe.io/api/v1/${name}`);
+  $('#nameid').attr('placeholder',`${name}`);
+ });
+});
```

Comment:
You don't need JQuery for this. It can be done pretty easily using JavaScript.

```suggestion
document.addEventListener('DOMContentLoaded', () => {
  const button_node = document.querySelector('button');

  button_node.addEventListener('click', () => {
    const name_node = document.querySelector('#nameid');
    const image_node = document.querySelector('.image');
    const name = name_node.value;

    image_node.setAttribute('src', `https://joeschmoe.io/api/v1/${name}`);
    name_node.setAttribute('placeholder', name);
  });
});
```
(I didn't test above code. So, make sure you test it at least once locally if you add it in your project)